src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs

```diff
@@ -66,6 +67,7 @@ public static partial class PlatformDetection
         public static bool IsNetNative { get { throw null; } }
         public static bool IsNonZeroLowerBoundArraySupported { get { throw null; } }
         public static bool IsNotIntMaxValueArrayIndexSupported { get { throw null; } }
+        public static bool IsNotArgIteratorSupported { get { throw null; } }
```

Comment:
Wouldn't `IsArgIteratorNotSupported` read better? Currently it seems to say that something called `NotArgIterator` is supported?